ccl/changefeedccl: TestAlterChangefeedPersistSinkURI failed

ccl/changefeedccl.TestAlterChangefeedPersistSinkURI [failed](https://tanzanite.cluster.engflow.com/invocations/default/78964d4c-58d5-4306-a532-70af90fa4fcb?testReportRun=13&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [7bb52a7d1c75d5adfdfa53e5fcff6f5e6497408f](https://github.com/cockroachdb/cockroach/commits/7bb52a7d1c75d5adfdfa53e5fcff6f5e6497408f):


```
=== RUN   TestAlterChangefeedPersistSinkURI
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestAlterChangefeedPersistSinkURI3223717628
    test_log_scope.go:81: use -show-logs to present logs inline
    helpers_test.go:1321: condition failed to evaluate within 45s: from helpers_test.go:1326: Expected status:paused but found status:pause-requested
    panic.go:523: -- test log scope end --
test logs left over in: outputs.zip/logTestAlterChangefeedPersistSinkURI3223717628
--- FAIL: TestAlterChangefeedPersistSinkURI (46.11s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=13</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedPersistSinkURI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-36275

--- a/pkg/ccl/changefeedccl/alter_changefeed_test.go
+++ b/pkg/ccl/changefeedccl/alter_changefeed_test.go
@@ -710,7 +710,11 @@ func TestAlterChangefeedPersistSinkURI(t *testing.T) {
 	const unredactedSinkURI = "null://blah?AWS_ACCESS_KEY_ID=the_secret"
 
 	ctx := context.Background()
-	srv, rawSQLDB, _ := serverutils.StartServer(t, base.TestServerArgs{})
+	srv, rawSQLDB, _ := serverutils.StartServer(t, base.TestServerArgs{
+		Knobs: base.TestingKnobs{
+			JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals(),
+		},
+	})
 	defer srv.Stopper().Stop(ctx)
 
 	s := srv.ApplicationLayer()

